keys/employees-keys/artur-kowalski-key.asc

```diff
@@ -0,0 +1,64 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
@arturkow2000 The key name should follow the `your-name-key-signed.asc` scheme according to https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/employees-keys/README.md#create-a-pull-request-with-key

Reply:
Fixed.